setup.py

```diff
@@ -40,7 +40,7 @@
     'henet': ['beautifulsoup4'],
     'hetzner': ['dnspython>=1.15.0','beautifulsoup4'],
     'easyname': ['beautifulsoup4'],
-    'localzone': ['localzone'],
```

Comment:
It is not necessary to remove the comma here. Python supports arrays with trailing comma, and it allows to have cleaner diff in GIT when a new value is added: indeed, instead of having two lines modified (the new one, and the previous one with an added comma), only one line is modified, the one that is added.